tests/validation/validate_ball_and_stick.hpp

```diff
@@ -50,7 +50,7 @@ void run_ncomp_convergence_test(
                 seg->set_compartments(ncomp);
             }
         }
-        hw::node nd(1, backend==backend_kind::gpu? 1: 0);
+        hw::node_info nd(1, backend==backend_kind::gpu? 1: 0);
```

Comment:
To me it is clearer, because I am explicitly stating that the result of the expression is an integer, whereas `a==b` relies on the implicit conversion of `bool` to unsigned.